modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/issue/IssueResource.java

```diff
@@ -479,21 +483,34 @@ private UpdateIssueParams generateUpdateIssueParams( final UpdateIssueJson json
 
     private PrincipalKeys filterInvalidAssignees( final List<PrincipalKey> assignees )
     {
-        return PrincipalKeys.from( assignees.stream().filter( this::isValidAssignee ).collect( Collectors.toList() ) );
+        return PrincipalKeys.from( assignees.
+            stream().
+            filter( this::isValidAssignee ).
+            collect( Collectors.toList() ) );
     }
 
+
     private boolean isValidAssignee( final PrincipalKey principalKey )
     {
-        final PrincipalKeys membershipKeys = securityService.getAllMemberships( principalKey );
-        if ( membershipKeys.getSize() > 0 )
-        {
-            final Principals memberships = securityService.getPrincipals( membershipKeys );
-            return memberships.stream().anyMatch( this::hasIssuePermissions );
-        }
-        else
+        if ( principalKey.isUser() )
         {
-            return false;
+            final ProjectName projectName = ProjectName.from( ContextAccessor.current().getRepositoryId() );
+            final PrincipalKeys membershipKeys = securityService.getAllMemberships( principalKey );
+
+            if ( ProjectConstants.DEFAULT_PROJECT_NAME.equals( projectName ) )
+            {
+                return membershipKeys.stream().anyMatch( this::hasManagerAccess );
+            }
+            else
+            {
+                final ProjectPermissions projectPermissions = projectService.getPermissions( projectName );
+
+                return isProjectOwnerOrEditor( principalKey, projectPermissions ) || membershipKeys.stream().
```

Comment:
This should be (almost) symmetric how default project and other projects handle permissions. Only one exception for now with Contributor role